core/runtime/wasm_edge/module_factory_impl.cpp

```diff
@@ -391,14 +391,20 @@ namespace kagome::runtime::wasm_edge {
   }
 
   CompilationOutcome<void> ModuleFactoryImpl::compile(
-      std::filesystem::path path_compiled, BufferView code) const {
+      std::filesystem::path path_compiled,
+      BufferView code,
+      const RuntimeContext::ContextParams &config) const {
     if (config_.exec == ExecType::Interpreted) {
       OUTCOME_TRY(writeFileTmp(path_compiled, code));
       return outcome::success();
     }
     OUTCOME_TRY(configure_ctx, configureCtx());
     WasmEdge_ConfigureCompilerSetOptimizationLevel(
         configure_ctx.raw(), WasmEdge_CompilerOptimizationLevel_O3);
+    if (config.wasm_ext_bulk_memory) {
```

Comment:
Please check (create wasm with bulk ops, or check wasmedge source code) if `loadCompiled` works.
If it doesn't work then `config` should be passed to `configureCtx`